website/data/features/api.mdx

```diff
@@ -1,4 +1,4 @@
-<img alt="Features" src="/img/feature-api.svg" />
+<img alt="" src="/img/feature-api.svg" />
```

Comment:
Is there any reason to not have actual alt text for these icons? They should probably also be links to the relevant sections

```suggestion
<a href="/docs/api/spec"><img alt="FDC3 API Part" src="/img/feature-api.svg" /></a>
```